spec/resource_spec.rb

```diff
@@ -249,7 +249,7 @@
       let(:timestamp) { '2015-01-02'.to_time(:utc) }
       let(:answers) do
         {
-          "[question(3), option(\"10021-other\")]" => "Some other text field answer",
+          "[question(3), option(\\\"10021-other\\\")]" => "Some other text field answer",
```

Comment:
it might be minorly easier to read as

`'[question(3), option(\"10021-other\")]'`

(single quotes will prevent interpolation)